release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -921,6 +926,16 @@ Support for deploying custom schedulers manually has been removed with this rele
 
 Support for deploying {sno} clusters with OpenShiftSDN has been removed with this release. OVN-Kubernetes is the default networking solution for {sno} deployments.
 
+==== Removal of Jenkins images from install payload
+
+* {product-title} 4.11 moves the "OpenShift Jenkins" and "OpenShift Agent Base" images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`. For more information, see xref:../cicd/jenkins/openshift-jenkins.adoc#openshift-jenkins[OpenShift Jenkins].
+
+* {product-title} 4.11 removes "OpenShift Jenkins Maven" and "NodeJS Agent" images from its payload. Previously, {product-title} 4.10 deprecated these images. Red Hat no longer produces these images, and they are not available from the `ocp-tools-4` repository at `registry.redhat.io`.
++
+However, upgrading to {product-title} 4.11 does not remove "OpenShift Jenkins Maven" and "NodeJS Agent" images from 4.10 and earlier releases. And Red Hat provides bug fixes and support for these images on through the end of the 4.10 release lifecycle, in accordance with the link:https://access.redhat.com/support/policy/updates/openshift[{product-title} lifecycle policy].
++
+For more information, see xref:../cicd/jenkins/openshift-jenkins.adoc#openshift-jenkins[OpenShift Jenkins].
```

Comment:
> to try and minimize the freak out @coreydaley @akram @rgormley I think we should also clarify here that a) the jenkins maven and nodejs imagestreams installed from 4.10 and earlier will not be removed if you upgrade to 4.11, b) openshift engineering will support those 4.10 etc. images on 4.11+ cluster through the end of the 4.10 maintenance window

@gabemontero @coreydaley @akram @rgormley PTAL at this updated version.

Reply:
That looks good to me. Thanks!